src/platform/nrfconnect/wifi/NrfWiFiDriver.h

```diff
@@ -39,6 +39,9 @@ class NrfScanResponseIterator : public Iterator<WiFiScanResponse>
 class NrfWiFiDriver final : public WiFiDriver
 {
 public:
+    static constexpr const char * kSsidKey = "g/wi/s";
```

Comment:
I guess I lack background on how Zephyr settings works. @Damian-Nordic could you please explain a bit why these particular keys were chosen?

Reply:
I tried to keep the convention used by other Matter components using `DefaultStorageKeyAllocator`. `g/` is a common prefix for Matter settings that are not fabric-scoped. "wi" is just my invention to organize Wi-Fi-related keys.

Reply:
I'll add a comment about this `DefaultStorageKeyAllocator` inspiration.